library/models.py

```diff
@@ -54,6 +55,11 @@ def __unicode__(self):
     def get_absolute_url(self):
         return reverse('library:book_detail', kwargs={'pk': self.pk})
 
+    @property
+    def index_strings(self):
+        return [self.title, self.isbn, self.authors, self.subtitle,
```

Comment:
Good question :)
But, out of curiosity, why a tuple? :)
(I've moved to tuples, anyway.)